frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -130,6 +130,17 @@ class MediaManagement extends Component {
                         isAdvanced={true}
                         size={sizes.MEDIUM}
                       >
+                        <FormLabel>Use Reflinks instead of Copy</FormLabel>
+
+                        <FormInputGroup
+                          type={inputTypes.CHECK}
+                          name="copyUsingReflinks"
+                          helpText="Use Reflinks when trying to copy files from torrents that are still being seeded"
+                          helpTextWarning="Reflinks are currently only implemented for Linux."
```

Comment:
While you're in here should we make this a bit more descriptive and list the filesystems on linux where it works?

Reply:
Done. More and more filesystems are adding support, because of the obvious features. ZFS can do it too, but ZFS and Linux is still a bit weird.